Formula/gemstash.rb

```diff
@@ -0,0 +1,124 @@
+class Gemstash < Formula
+  desc "RubyGems.org cache and private gem server"
+  homepage "https://github.com/bundler/gemstash"
+  url "https://rubygems.org/downloads/gemstash-2.0.0.gem"
+  sha256 "511a5773c43507b913c4f2f57d33a3d972bd71378b2669c22a357b3351a82bf4"
+  head "https://github.com/bundler/gemstash.git"
+
+  if MacOS.version < :mojave
+    depends_on "openssl"
+    depends_on "ruby"
+  end
+
+  def install
+    if build.head?
+      system "gem", "build", "gemstash.gemspec"
+      gem_file = Dir["*.gem"].first
```

Comment:
This looks very errorprone. I think you should remove HEAD builds for the first version of this formula.